nautobot/extras/tests/test_customfields.py

```diff
@@ -157,6 +157,52 @@ def test_multi_select_field(self):
         # Delete the custom field
```

Comment:
Looking at the other test cases in this class, above, it seems wrong to me they *aren't* calling `clean()` or `validated_save()`, but are instead just calling `save()`, bypassing the cleaning logic altogether. It seems to me like it might be better to update them so that they all use `validated_save()`?

Reply:
Yeah I was wondering that as well. I'll go ahead and change them all to validated_save where appropriate.